test/unit_tests/arkode/CXX_serial/ark_butcher_tests.cpp

```diff
@@ -0,0 +1,173 @@
+/*---------------------------------------------------------------
+ Programmer(s): Daniel R. Reynolds @ SMU
+ ----------------------------------------------------------------
+ Copyright (c) 2022, Southern Methodist University.
+ All rights reserved.
+ For details, see the LICENSE file.
+ ----------------------------------------------------------------
+ Routine to test all built-in Butcher tables to check their
+ order of accuracy.
+---------------------------------------------------------------*/
+
+// Header files
+#include <arkode/arkode.h>
+#include <arkode/arkode_butcher_erk.h>
+#include <arkode/arkode_butcher_dirk.h>
+#include <sundials/sundials_types.h>
+#include <vector>
+#include <iostream>
+#include <ostream>
+#include <string>
+
+// Main Program
+int main() {
+
+  // set vectors of individual tables to test
+  std::vector<ARKODE_ERKTableID> Tables_ERK = {ARKODE_HEUN_EULER_2_1_2,
+    ARKODE_BOGACKI_SHAMPINE_4_2_3, ARKODE_ARK324L2SA_ERK_4_2_3, ARKODE_ZONNEVELD_5_3_4,
+    ARKODE_ARK436L2SA_ERK_6_3_4, ARKODE_SAYFY_ABURUB_6_3_4, ARKODE_CASH_KARP_6_4_5,
+    ARKODE_FEHLBERG_6_4_5, ARKODE_DORMAND_PRINCE_7_4_5, ARKODE_ARK548L2SA_ERK_8_4_5,
+    ARKODE_VERNER_8_5_6, ARKODE_FEHLBERG_13_7_8, ARKODE_ARK437L2SA_ERK_7_3_4,
+    ARKODE_ARK548L2SAb_ERK_8_4_5};
+  std::vector<std::string> STables_ERK = {"HEUN_EULER_2_1_2", "BOGACKI_SHAMPINE_4_2_3",
+    "ARK324L2SA_ERK_4_2_3", "ZONNEVELD_5_3_4", "ARK436L2SA_ERK_6_3_4", "SAYFY_ABURUB_6_3_4",
+    "CASH_KARP_6_4_5", "FEHLBERG_6_4_5", "DORMAND_PRINCE_7_4_5", "ARK548L2SA_ERK_8_4_5",
+    "VERNER_8_5_6", "FEHLBERG_13_7_8", "ARK437L2SA_ERK_7_3_4", "ARK548L2SAb_ERK_8_4_5"};
+  std::vector<ARKODE_DIRKTableID> Tables_DIRK = {ARKODE_SDIRK_2_1_2,
+    ARKODE_BILLINGTON_3_3_2, ARKODE_TRBDF2_3_3_2, ARKODE_KVAERNO_4_2_3,
+    ARKODE_ARK324L2SA_DIRK_4_2_3, ARKODE_CASH_5_2_4, ARKODE_CASH_5_3_4,
+    ARKODE_SDIRK_5_3_4, ARKODE_KVAERNO_5_3_4, ARKODE_ARK436L2SA_DIRK_6_3_4,
+    ARKODE_KVAERNO_7_4_5, ARKODE_ARK548L2SA_DIRK_8_4_5,
+    ARKODE_ARK437L2SA_DIRK_7_3_4, ARKODE_ARK548L2SAb_DIRK_8_4_5,
+    ARKODE_SDIRK2_2_1_2, ARKODE_ESDIRK324L2SA_4_2_3, ARKODE_ESDIRK325L2SA_5_2_3,
```

Comment:
`ARKODE_SDIRK2_2_1_2` is not a valid method name.  I think it can be removed since `ARKODE_SDIRK_2_1_2` is already the first item in vector

Reply:
How in the world did this test even compile and run?  No matter; I've fixed it now.  Thanks for the catch!